src/http/web_socket/protocol.cr

```diff
@@ -285,4 +292,12 @@ class HTTP::WebSocket::Protocol
 
     raise ArgumentError.new("No host or path specified which are required.")
   end
+
+  def self.key_challenge(key)
```

Comment:
Assume this is public because is used above (`Protocol.key_challenge`). In that case, perhaps will be better add a `:nodoc:` marker to avoid this showing up in the documentation?

Thank you.

Reply:
Yes, this needs to be accessible from `WebSocketHandler`.
The entire `Protocol` class is just internal implementation and not documented.

Reply:
Ah, missed that from the PR diff, thank you for your response!